addons/web_editor/static/src/components/html_history_dialog/html_history_dialog.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="web_editor.HtmlHistoryDialog" owl="1">
+        <Dialog size="size" title="title">
+            <div class="dialog-container html-history-dialog">
+                <div class="revision-list d-flex flex-column align-content-stretch">
+                    <t t-if="!state.revisionsData.length">
+                        <div class="text-center w-100 pb-2 pt-0 px-0 fw-bolder">No history</div>
+                    </t>
+                    <t t-foreach="state.revisionsData" t-as="rec" t-key="rec.id">
+                        <a type="object" href="#" role="button"
+                           t-attf-class="btn btn-outline-primary #{state.revisionId === rec.id ? 'active' : ''}"
+                           t-on-click="() => this.updateCurrentRevision(rec.id)">
+                            <strong><t t-esc="this.getRevisionDate(rec)" /></strong><br/>
+                            <small><t t-esc="this.getRevisionAuthor(rec)" /></small>
+                        </a>
+                    </t>
+                </div>
+                <div class="history-container o_notebook">
+                    <ul class="nav nav-tabs" role="tablist">
+                        <li class="nav-item" role="presentation">
+                            <button class="nav-link active"
+                                    id="history-content"
+                                    data-bs-toggle="tab" data-bs-target="#history-content-tab"
+                                    type="button" role="tab"
+                                    aria-controls="content" aria-selected="true">Content</button>
```

Comment:
Omg, even in the html ! x) Except in Python where I'm not sure, I don't think this is valid linting in any language:
```
start part1
    // Randomly indented to match part1 rather than one tab length.
    part2 end
```
nor this:
```
start
    middle end
```
Usually it's more like:
```
start
    part1
    part2
end
```
Except in very specific cases, like this one for example:
```js
const myVar =
    condition && key in quiteLongVariable
        ? ternaryTrue
        : ternaryFalse;
```
But that's because the only way to "close" the statement is the semicolon, and you don't want to write a single line with just one semicolon to close it at the right indentation. When you have a way of properly closing the statement, either a closing `}` or a `</tag>` then it should be the closing line.

HTML has one particular drawback with that in the fact that depending on where you are, spaces may matter. In that case I'd advocate for this format:
```html
<span
    attr1="..."
    attr2="..."
    attr3="..."
>Inline content in which spaces matter.</span>
```
This way you gain the vertical space you need for all the arguments and you keep the closing at the right indentation length. To be honest, in this particular case, I have no idea if this is a commonly used format or not. It's just my personal preference, but honestly I don't see any other way that could work because you need the `>` to be on the same line as the text for technical html reasons, and it can't also be on the same line as `<span` because it has too much attributes.

Now that being said, writing this actually made me wonder. Do people write multiple attributes on one line ? I think most examples I have seen were of the above form, with one attribute by line, not including the tag line.

Reply:
Except the inconsistent number of attribute per line ( on my part)
The alignment is from the formater : [see here](https://i.imgur.com/izTuKmW.gifv)